translations/config.json

```diff
@@ -90,7 +90,8 @@
         "e-alfred ocdownloader",
         "e-alfred flowupload",
         "e-alfred nextcloud-scanner",
-        "affan98 forms"
+        "affan98 forms",
```

Comment:
@jancborchardt @rullzer  I suddenly realise we should rename this to nextcloud/forms, right?

Reply:
Yup! The repository is moved and https://github.com/affan98/forms redirects to https://github.com/nextcloud/forms :)